components/drops/utils.ts

```diff
@@ -28,7 +28,10 @@ export function formatDropStartTime(
     })
   }
 
-  return intlFormat(startTime, options, { locale })
+  return intlFormat(startTime, { ...options, month: 'long' }, { locale })
+    .split(' ')
+    .reverse()
+    .join('. ')
```

Comment:
Please do not format the date by operating the string, because the format follows user's locale and also are different.
<img width="707" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/96c92064-6f5b-4fa2-867a-d3d934d64ce4">


``` suggestion
  return intlFormat(startTime, { ...options, month: 'long' }, { locale })
```

Reply:
what you should change is line 22

<img width="642" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/20ae8f23-830b-4701-aad3-162cbc62b8ef">

Reply:
yup, oki